tempest2/src/main/kotlin/app/cash/tempest2/internal/DynamoDbLogicalDb.kt

```diff
@@ -124,20 +132,42 @@ internal class DynamoDbLogicalDb(
 
     override fun batchLoadAsync(
       keys: KeySet,
-      consistentReads: Boolean
+      consistentReads: Boolean,
+      maxPageSize: Int
     ): Publisher<ItemSet> {
-      val (requests, requestsByTable, batchRequest) = toBatchLoadRequest(keys, consistentReads)
-      return dynamoDbEnhancedClient.batchGetItem(batchRequest)
+      // TODO: Replace this with publisher stream concatenation
```

Comment:
suggest filing a github issue and linking that inline here with the TODO. can also attach comments to the associated diabled tests for easier tracking of this concern.